src/app/story/directives/status.component.ts

```diff
@@ -1,64 +1,43 @@
       <dd>
-        <span [class]="statusClass">{{statusText}}</span>
-        <ng-container *ngIf="isPublished">
-          <button *ngIf="editStatus" class="btn-link edit-status" (click)="toggleEdit()">Hide</button>
-          <button *ngIf="!editStatus" class="btn-link edit-status" (click)="toggleEdit()">Edit</button>
-          <prx-button *ngIf="editStatus" [model]="story" visible=1 orange=1 disabled=0
-            [working]="isPublishing" (click)="togglePublish()">Unpublish</prx-button>
-        </ng-container>
+        <select *ngIf="currentStatus !== 'published'; else published"
+          [class.changed]="nextStatus !== currentStatus"
+          [ngModel]="nextStatus"
+          (ngModelChange)="statusChange($event)">
+          <option *ngFor="let status of statusOptions"
+                  [value]="status">{{status | titlecase}}</option>
+        </select>
+        <ng-template #published>Published</ng-template>
       </dd>
 
-      <dt *ngIf="isPublished && isScheduled">Publishing</dt>
-      <dt *ngIf="isPublished && !isScheduled">Published</dt>
-      <dd *ngIf="isPublished"><p>{{story.publishedAt | date:"short"}}</p></dd>
-
-      <dt *ngIf="isReleased && !isPublished">Release</dt>
-      <dd *ngIf="isReleased && !isPublished"><p>{{story.releasedAt | date:"short"}}</p></dd>
-
-      <dt>Saved</dt>
-      <dd *ngIf="!id"><p>Not Saved</p></dd>
-      <dd *ngIf="id"><p *ngIf="story?.updatedAt">{{story.updatedAt | date:"short"}}</p></dd>
+      <dt>Last Saved</dt>
+      <dd *ngIf="!id">Not Saved</dd>
+      <dd *ngIf="id">{{story.updatedAt | date: 'short'}}</dd>
 
-      <dt>Progress</dt>
-      <dd *ngIf="!id">
-        <p *ngIf="changed && !normalInvalid">Ready to create</p>
-        <p *ngIf="changed && normalInvalid" class="error">Unable to create</p>
-        <button *ngIf="changed && normalInvalid" class="btn-link"
-          (click)="showProblems()">{{normalInvalidCount}}</button>
-      </dd>
-      <dd *ngIf="id">
-        <ng-container *ngIf="strictInvalid">
-          <p *ngIf="isPublished || normalInvalid" class="error">Invalid episode</p>
-          <p *ngIf="notPublished && !normalInvalid">Not ready to publish</p>
-          <button (click)="showProblems()" class="btn-link">{{strictInvalidCount}}</button>
-        </ng-container>
-        <ng-container *ngIf="notPublished && !strictInvalid">
-          <p *ngIf="changed">Ready after save</p>
-          <p *ngIf="!changed">Ready to publish</p>
-          <prx-button [model]="story" visible=1 orange=1 [disabled]="changed"
-            [working]="isPublishing" (click)="togglePublish()">Publish</prx-button>
-        </ng-container>
-        <ng-container *ngIf="isPublished && !strictInvalid">
-          <p *ngIf="changed">Unsaved changes</p>
-          <p *ngIf="!changed">Complete</p>
-        </ng-container>
+      <dt>Dropdate</dt>
+      <dd>
+        <prx-tz-datepicker *ngIf="nextStatus !== 'published' || currentStatus === 'published'; else publishImmediately"
```

Comment:
So this is probably the only UI thing I feel strongly about in here... but what if we throw an "edit / change" toggle link on this datepicker?  Like the one that used to be next to the status:

![image](https://user-images.githubusercontent.com/1410587/59375099-a7728380-8d0a-11e9-9965-f666bae7ef31.png)

It just feels a bit busy to always show these 3 fields, when you're most often not changing it.

Reply:
Also found a bug here (which may be upstream)...

1. Publish an episode (doesn't happen when it's unpublished)
2. Change any field of the datepicker
3. Click "discard"
4. See the error: 
    ```
    ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'changed: false'. Current value: 'changed: true'.
    ```

I also notice that when you publish the episode, and change the field and change it back, it still has the "changed" outline on it.  (Probably related).

Reply:
Do you mean an edit button next to the dropdate field? If so, I would be okay with that.

Reply:
Yep, just to make things a little less "busy" when the episode is published.  (When you probably won't be editing the drop date).

Reply:
I will look at that date discard bug. Good find.

I do feel like those three stacked fields make the box a little tall. I started to worry about screen resolution with the way that's fixed position.

Reply:
Good point.  Is there some chance this field would be unusable on a phone?

Reply:
Fits on my iPhone X but we definitely dont have a lot of extra space in there.
![IMG_0850](https://user-images.githubusercontent.com/2250413/59387419-c420b280-8d2e-11e9-976a-c54bff4192c4.PNG)

Reply:
Have to come up with a solution yet, but I at least see what happening a little better on this date discard bug. It's related to showing the publishedAt vs setting the releasedAt. The date picker emits a change after the discard which then sets releasedAt, when we were looking at publishedAt for published episodes, so that underlying swap seems to cause this error. So... working on it!

Reply:
Would it be helpful to just set a `publishedReleasedAt` virtual field on the Model... that just "does the right thing" on encode/decode?  I guess it gets a bit complicated since publish/unpublish are standalone POST actions.

Reply:
I fixed that date discard bug in the underlying datepicker in styleguide 2.1.12 https://github.com/PRX/styleguide.prx.org/pull/190 to not emit changes when the date value does not change. Seemed like we were getting extra emits there anyway.

Reply:
+1 works great now